projects/genius-yield/index.js

```diff
@@ -1,17 +1,56 @@
+const INDY = '533bb94a8850ee3ccbe483106489399112b74c905342cb1792a797a0' + '494e4459'
+const iUSD = 'f66d78b4a3cb3d37afa0ec36461e51ecbde00f26c8f0a68f94b69880' + '69555344'
+const JELLY = '5c1c91a65bedac56f245b8184b5820ced3d2f1540e521dc1060fa683' + '4a454c4c59'
+const JPG = '681b5d0383ac3b457e1bcc453223c90ccef26b234328f45fa10fd276' + '4a5047'
+const LENFI = '8fef2d34078659493ce161a6c7fba4b56afefa8535296a5743f69587' + '41414441'
+const LIFI = '7914fae20eb2903ed6fd5021a415c1bd2626b64a2d86a304cb40ff5e' + '4c494649'
+const LQ = 'da8c30857834c6ae7203935b89278c532b3995245295456f993e1d24' + '4c51'
+const MELD = '6ac8ef33b510ec004fe11585f7c5a9f0c07f0c23428ab4f29c1d7d10' + '4d454c44'
+const MILK = '8a1cfae21368b8bebbbed9800fec304e95cce39a2a57dc35e2e3ebaa' + '4d494c4b'
+const MIN = '29d222ce763455e3d7a09a665ce554f00ac89d2e99a1a83d267170c6' + '4d494e'
+const NEWM = '682fe60c9918842b3323c43b5144bc3d52a23bd2fb81345560d73f63' + '4e45574d'
+const NINJAZ = 'df1d850c46d6c9d12cbf6181c35db9225a91b77c8a646b7f636f8ae4' + '0014df104e494e4a415a'
+const NMKR = '5dac8536653edc12f6f5e1045d8164b9f59998d3bdc300fc92843489' + '4e4d4b52'
+const NTX = 'edfd7a1d77bcb8b884c474bdc92a16002d1fb720e454fa6e99344479' + '4e5458'
+const OPT = '1ddcb9c9de95361565392c5bdff64767492d61a96166cb16094e54be' + '4f5054'
+const OPTIM = 'e52964af4fffdb54504859875b1827b60ba679074996156461143dc1' + '4f5054494d'
+const RJV = '8cfd6893f5f6c1cc954cec1a0a1460841b74da6e7803820dde62bb78' + '524a56'
+const SHEN = '8db269c3ec630e06ae29f74bc39edd1f87c819f1056206e879a1cd61' + '5368656e4d6963726f555344'
+const SNEK = '279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f' + '534e454b'
+const SOC = '25f0fc240e91bd95dcdaebd2ba7713fc5168ac77234a3d79449fc20c' + '534f4349455459'
+const SUNDAE = '9a9693a9a37912a5097918f97918d15240c92ab729a0b7c4aa144d77' + '53554e444145'
+const VYFI = '804f5544c1962a40546827cab750a88404dc7108c0f588b72964754f' + '56594649'
+const WMT = '1d7f33bd23d85e1a25d87d86fac4f199c3197a2f7afeb662a0f34e1e' + '776f726c646d6f62696c65746f6b656e'
+const WRT = 'c0ee29a85b13209423b10447d3c2e6a50641a15c57770e27cb9d5073' + '57696e67526964657273'
 
-const owner = 'addr1w8r99sv75y9tqfdzkzyqdqhedgnef47w4x7y0qnyts8pznq87e4wh'
+const stakingAddr = 'addr1w8r99sv75y9tqfdzkzyqdqhedgnef47w4x7y0qnyts8pznq87e4wh'
+const dexPaymentCredential = 'edfff663d37fc5f9753bc4222e0da2bfe08aa48db0837d2c329adeb3'
 
 module.exports = {
   timetravel: false,
   cardano: {
-    staking: sumTokensExport({ owner, tokens: [gens, nmkr, ntx, emp, gensx]}),
-    tvl: () => ({})
+    staking: sumTokensExport({ stakingAddr, tokens: [GENS, NMKR, NTX, EMP, GENSX]}),
+    tvl: sumTokensExport({ dexPaymentCredential, tokens: [AGIX, BOOK, CGV, CHRY, cNETA, COPI, DJED, EMP, ENCS, FACT, FLAC, GENS, GENSX, GXLA, HOSKY, HUNT, IAG, iBTC, iETH, INDY, iUSD, JELLY, JPG, LENFI, LIFI, LQ, MELD, MILK, MIN, NEWM, NINJAZ, NMKR, NTX, OPT, OPTIM, RJV, SHEN, SNEK, SOC, SUNDAE, VYFI, WMT, WRT]})
```

Comment:
ADA is still missing.

There is also ADA locked into the DEX. Not only Cardano Native tokens, but also ADA.

Reply:
Added 'lovelace' in d4c82ba0fa2ef67b5b89ab8558db975a6355dbdd to include ADA as well in the TVL.